recipes/php.rb

```diff
@@ -34,3 +34,8 @@
   user node['nginx']['user']
   group node['nginx']['group']
 end
+
+execute 'run laravel migrations to setup database' do
+  command ['php', 'artisan', 'migrate', '--force']
```

Comment:
I might consider specifying the full path of `php` or using the `environment` property to set the `PATH` for this command.